src/controllers/hid/hidiooutputreport.cpp

```diff
@@ -78,6 +80,7 @@ bool HidIoOutputReport::sendCachedData(QMutex* pHidDeviceAndPollMutex,
         hid_device* pHidDevice,
         const mixxx::hid::DeviceInfo& deviceInfo,
         const RuntimeLoggingCategory& logOutput) {
+    Q_UNUSED(deviceInfo);
```

Comment:
Please remove the unused parameter from this function, instead of silencing the warning.